src/renderer/waitDialog/css/message.css

```diff
@@ -31,7 +31,7 @@
     padding: .5em;
 }
 
-.spinner {
+.animate .spinner {
```

Comment:
We have followed a convention where the "action" class placed last. Here we'd use `.spinner .animate` instead.

Reply:
I add the `animate` class to the body, so `.animate` is a parent of `.spinner`.

(I could have added the class to the spinner, but that would increase the coupling)